pandas/io/gbq.py

```diff
@@ -224,13 +268,25 @@ def run_query(self, query, verbose=True):
 
         while not query_reply.get('jobComplete', False):
             if verbose:
-                print('Waiting for job to complete...')
+                self.print_elapsed_seconds('  Elapsed', 's. Waiting...')
             try:
                 query_reply = job_collection.getQueryResults(projectId=job_reference['projectId'],
                                                              jobId=job_reference['jobId']).execute()
             except HttpError as ex:
                 self.process_http_error(ex)
 
+        if verbose:
+            if query_reply['cacheHit']:
+                print('Query done.\nCached result.\nPrice: 0 $.\n')
+            else:
+                # Charges are rounded to the nearest MB, with a minimum 10 MB data processed per table referenced by the query.
+                bytes_processed = int(query_reply.get('totalBytesProcessed', '0'))
```

Comment:
price info is here not 100% precise. And it is sure that google will change billing methods and rates. But this info is helpful I think.

Can you explain why you do not want to see price here?